apps/coscrad-frontend/src/components/NavBar/NavBarItem.spec.tsx

```diff
@@ -0,0 +1,20 @@
+import { MemoryRouter } from 'react-router-dom';
+import { renderWithProviders } from '../../utils/test-utils';
+import { NavItemInfo } from './NavBar';
+import { NavBarItem } from './NavBarItem';
+
+const dummyNavBarItemProps: NavItemInfo = {
+    link: 'http://samplesite.com',
```

Comment:
Make the dummy URL https so SonarCloud won't complain.

Reply:
done